### PR TITLE
Use bloop-rifle module from scala-cli/bloop-core repo

### PR DESCRIPTION
**Pull request by alexarchambault**
This removes the `bloop-rifle` module from the Scala CLI repo, and uses the one from [scala-cli/bloop-core](https://github.com/scala-cli/bloop-core) instead, where it lives as a substitute to the `launcher` and `bloopgun` modules of Bloop mainline.

### CODE REVIEW

**Comment by alexarchambault** [comment]
Needs https://github.com/scala-cli/bloop-core/pull/139

File: project/deps.sc [hunk 1]
```diff
@@ -95,9 +95,9 @@ object Deps {
   // that Scala CLI supports.
   def ammonite = ivy"com.lihaoyi:::ammonite:2.5.6-1-f8bff243"
   def asm      = ivy"org.ow2.asm:asm:9.4"
-  def bloop    = ivy"io.github.alexarchambault.bleep:bloop-frontend_2.12:1.5.6-sc-4"
   // Force using of 2.13 - is there a better way?
   def bloopConfig      = ivy"ch.epfl.scala:bloop-config_2.13:1.5.5"
+  def bloopRifle       = ivy"io.github.alexarchambault.bleep:bloop-rifle_2.13:1.5.6-sc-6"
```
Comment (Gedochao) [inline project/deps.sc:100]:
```suggestion
  def bloopRifle       = ivy"io.github.alexarchambault.bleep:bloop-rifle_2.13:1.5.6-sc-7"
```

**Review by Gedochao — COMMENTED** [review]
NIT

**Review by Gedochao — APPROVED** [review]

**Review by tgodzik — APPROVED** [review]
LGTM

**Pull request merged**